src/nft/components/collection/Sweep.tsx

```diff
@@ -215,8 +215,8 @@ export const Sweep = ({ contractAddress, minPrice, maxPrice }: SweepProps) => {
           const poolPrice = calcPoolPrice(asset, isNFTX ? counterNFTX : counterNFT20)
           if (BigNumber.from(poolPrice).gt(0)) {
             isNFTX ? counterNFTX++ : counterNFT20++
-            asset.priceInfo.ETHPrice = poolPrice
           }
+          asset.priceInfo.ETHPrice = poolPrice
```

Comment:
so, previously, the price would be set to pool price only if its value was greater than zero, right? Now, it will be set regardless of the value

Reply:
yes

Reply:
no because the ones that are not sus are the only ones that are selectable and thus should be counted